app/routes/AccountDisplay/AccountDisplay.js

```diff
@@ -26,12 +29,14 @@ class AccountDisplay extends Component {
 		this.actions = [
 			{
 				label: "Log out",
-				onPress: this.userLogout,
+				onPress: this.userLogout.bind(this),
```

Comment:
Le `bind(this)` ne doit pas être utile comme on accède au store pour reset le token. Le contexte du component ne doit pas être essentiel dans ce cas